tutorials/inter_region_reports/README.md

```diff
@@ -236,17 +236,16 @@ other options for each VM.
e-north1-a],[australia-southeast1-a,europe-west1-c],[australia-southeast1-a,europe-west2-a],[australia-southeast1-a,europe-west3-a],[australia-southeast1-a,europe-west4-a],[australia-southeast1-a,europe-west6-a],[australia-southeast1-a,northamerica-northeast1-a],[australia-southeast1-a,southamerica-east1-a],[australia-southeast1-a,us-central1-a],[australia-southeast1-a,us-east1-b],[australia-southeast1-a,us-east4-a],[australia-southeast1-a,us-west1-a],[australia-southeast1-a,us-west2-a],[europe-north1-a,europe-west1-c],[europe-north1-a,europe-west2-a],[europe-north1-a,europe-west3-a],[europe-north1-a,europe-west4-a],[europe-north1-a,europe-west6-a],[europe-north1-a,northamerica-northeast1-a],[europe-north1-a,southamerica-east1-a],[europe-north1-a,us-central1-a],[europe-north1-a,us-east1-b],[europe-north1-a,us-east4-a],[europe-north1-a,us-west1-a],[europe-north1-a,us-west2-a],[europe-west1-c,europe-west2-a],[europe-west1-c,europe-west3-a],[europe-west1-c,europe-west4-a],[europe-west1-c,europe-west6-a],[europe-west1-c,northamerica-northeast1-a],[europe-west1-c,southamerica-east1-a],[europe-west1-c,us-central1-a],[europe-west1-c,us-east1-b],[europe-west1-c,us-east4-a],[europe-west1-c,us-west1-a],[europe-west1-c,us-west2-a],[europe-west2-a,europe-west3-a],[europe-west2-a,europe-west4-a],[europe-west2-a,europe-west6-a],[europe-west2-a,northamerica-northeast1-a],[europe-west2-a,southamerica-east1-a],[europe-west2-a,us-central1-a],[europe-west2-a,us-east1-b],[europe-west2-a,us-east4-a],[europe-west2-a,us-west1-a],[europe-west2-a,us-west2-a],[europe-west3-a,europe-west4-a],[europe-west3-a,europe-west6-a],[europe-west3-a,northamerica-northeast1-a],[europe-west3-a,southamerica-east1-a],[europe-west3-a,us-central1-a],[europe-west3-a,us-east1-b],[europe-west3-a,us-east4-a],[europe-west3-a,us-west1-a],[europe-west3-a,us-west2-a],[europe-west4-a,europe-west6-a],[europe-west4-a,northamerica-northeast1-a],[europe-west4-a,southamerica-east1-a],[europe-west4-a,us-central1-a],[europe-west4-a,us-east1-b],[europe-west4-a,us-east4-a],[europe-west4-a,us-west1-a],[europe-west4-a,us-west2-a],[europe-west6-a,northamerica-northeast1-a],[europe-west6-a,southamerica-east1-a],[europe-west6-a,us-central1-a],[europe-west6-a,us-east1-b],[europe-west6-a,us-east4-a],[europe-west6-a,us-west1-a],[europe-west6-a,us-west2-a],[northamerica-northeast1-a,southamerica-east1-a],[northamerica-northeast1-a,us-central1-a],[northamerica-northeast1-a,us-east1-b],[northamerica-northeast1-a,us-east4-a],[northamerica-northeast1-a,us-west1-a],[northamerica-northeast1-a,us-west2-a],[southamerica-east1-a,us-central1-a],[southamerica-east1-a,us-east1-b],[southamerica-east1-a,us-east4-a],[southamerica-east1-a,us-west1-a],[southamerica-east1-a,us-west2-a],[us-central1-a,us-east1-b],[us-central1-a,us-east4-a],[us-central1-a,us-west1-a],[us-central1-a,us-west2-a],[us-east1-b,us-east4-a],[us-east1-b,us-west1-a],[us-east1-b,us-west2-a],[us-east4-a,us-west1-a],[us-east4-a,us-west2-a],[us-west1-a,us-west2-a]]
```

Comment:
Too bad we're making you do this with the deprecation of those flags..

We used to have a way of creating zip and cross products across flags. Is there some support in YAML that we can use to create this list of lists?

If we can't do that, is there a bash command we can add to make this easier to generate?